### ISSUE TITLE
python2: ImportError: No module named error

### ISSUE THREAD

**Issue by blshkv** [JoMingyu/google-play-scraper#35]
The package claims python2 compatibility but I'm facing the following error message:
```
>>> Compiling source in /var/tmp/portage/dev-python/google-play-scraper-0.0.1.6/work/google-play-scraper-0.0.1.6 ...
 * python2_7: running distutils-r1_run_phase distutils-r1_python_compile
python2.7 setup.py build
Traceback (most recent call last):
  File "setup.py", line 3, in <module>
    from google_play_scraper import __version__
  File "/var/tmp/portage/dev-python/google-play-scraper-0.0.1.6/work/google-play-scraper-0.0.1.6/google_play_scraper/__init__.py", line 2, in <module>
    from urllib.error import HTTPError
ImportError: No module named error
 * ERROR: dev-python/google-play-scraper-0.0.1.6::pentoo failed (compile phase):
 *   (no error message)

```

It works fine with python3

**Comment by JoMingyu** [comment]
I'll fix it

**Comment by JoMingyu** [comment]
It fixed from version 0.0.1.8. I will setup CI with tox for Python 2 support perfectly.

**Comment by JoMingyu** [comment]
This issue tested by Python version 2.7.10. If this error reoccurs, reopen and tag me.
